sources/tools/Stride.TextureConverter/Backend/Wrappers/FINetWrapper/Classes/MetadataModels.cs

```diff
@@ -2790,14 +2779,12 @@ public TimeSpan? TimeStamp
             get
             {
                 FIURational[] stamp = GetTagArray<FIURational>("GPSTimeStamp");
-                if ((stamp == null) || stamp.Length != 3)
+                if (stamp is not { Length: 3 })
```

Comment:
I think this one is just more confusing than the previous one; the negative and positive is merged into the same comparison

Reply:
No problem , I think it is more readable but I understand that it can be confusing, so I reverted it 😀